### PR TITLE
Support TENDL 2021 in convert_tendl.py script

### PR DESCRIPTION
**Pull request by paulromano**
TENDL 2021 just came out on 12/30/2021. This PR adds support to convert the ACE files that are provided on the [TENDL 2021 website](https://tendl.web.psi.ch/tendl_2021/tendl2021.html).

### CODE REVIEW

File: convert_tendl.py [hunk 1]
```diff
@@ -92,6 +92,14 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
...
         'compressed_file_size': '2.3 GB',
         'uncompressed_file_size': '10.1 GB'
+    },
+    '2021': {
+        'base_url': 'https://tendl.web.psi.ch/tendl_2021/tar_files/',
+        'compressed_files': ['tendl21c.tar.bz2'],
+        'neutron_files': ace_files_dir.glob('tendl21c/*'),
+        'metastables': ace_files_dir.glob('tendl21c/*m'),
+        'compressed_file_size': '2.2 GB',
+        'uncompressed_file_size': '10.0 GB'
```
Comment (shimwell) [inline convert_tendl.py:102]:
```suggestion
        'uncompressed_file_size': '10.5 GB'
```

Comment (shimwell) [reply convert_tendl.py:102]: Perhaps being fussy here :-)

**Comment by shimwell** [comment]
Looks good, I am appreciating the easy of adding new libraries using the dictionary.

I have run this script with the new default ```-r``` argument set to 2021 and can confirm that

- [x] the neutron files download
- [x] the compressed file size is correctly stated
- [x] the uncompressed file size is correctly stated (close enough)
- [x] the ACE files are all converted to h5 files

I think this is ready to be merged

**Review by shimwell — APPROVED** [review]

**Comment by paulromano** [comment]
Thanks for taking a look @shimwell!

**Pull request merged**